### ISSUE TITLE
asdf which python -- unary operator expected

### ISSUE THREAD

**Issue by rlipscombe** [asdf-vm/asdf#447]
```
$ cd
$ asdf current python
system   (set by /home/roger/.tool-versions)
$ asdf which python
/home/roger/.asdf/lib/commands/which.sh: line 31: [: -eq: unary operator expected
```

```
$ cd src/foo
$ asdf current python
2.7.15   (set by /home/roger/src/foo/.tool-versions)
$ asdf which python
/home/roger/.asdf/installs/python/2.7.15/bin/python
```

Might be related to #191.

**Comment by Stratus3D** [comment]
@rlipscombe we just released v0.7.0, can you see if upgrading fixes this issue?

**Comment by Stratus3D** [comment]
Closing since I am not able to reproduce this issue in version 0.7.0.
